src/components/BooksList.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable arrow-parens */
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const BooksList = (state) => {
+  const mapProps = () => state.books.books.map((item) => (
+    <Book key={item} id={item.id} title={item.title} category={item.category} />
+  ));
+
+  return (
+    <div id="table-container">
+      <table>
+        <tr>
+          <th>Book ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </table>
+      {mapProps()}
+    </div>
+  );
+};
+
```

Comment:
Sorry man 🙏 we missed this in milestone 1.
kindly add the package and do the typechecking-props
![image](https://user-images.githubusercontent.com/31897434/76151980-c562ee00-60ba-11ea-90f4-43b11ac19e78.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896806/